src/layout/mutation.ts

```diff
@@ -74,17 +75,20 @@ async function process(): Promise<void> {
       switch (mutation.type) {
         case "attributes":
             if (task.shouldYield(timer)) { await task.suspend(timer); }
+            dom.regions(target as HTMLElement);
             processNode(target, Source.Attributes);
             break;
         case "characterData":
             if (task.shouldYield(timer)) { await task.suspend(timer); }
+            dom.regions(target as HTMLElement);
             processNode(target, Source.CharacterData);
             break;
         case "childList":
           // Process additions
           let addedLength = mutation.addedNodes.length;
           for (let j = 0; j < addedLength; j++) {
             let addedNode = mutation.addedNodes[j];
+            dom.regions(addedNode as HTMLElement);
```

Comment:
>regions [](start = 16, length = 7)

What does this do? Make this a verb, not a noun.

Reply:
Yes, I will make it findRegions and will resolve.

Reply:
Called it extractRegions.